action.yml

```
@@ -0,0 +1,31 @@
+name: Remote tflint Config
+description: Copies a tflint configuration file from a remote repo for local use
+author: ryanwholey
+inputs:
+  source-repo: 
+    required: true
+    description: The repository to copy a tflint configuration from
+  source-path:
+    description: The directory in the remote repo where the configuration exists
+    default: .
+  source-filename:
+    description: The source filename
+    default: .tflint.hcl
+  source-ref:
+    description: Ref or branch of the remote repository to target
+    default: main
```

```suggestion
    default: ''
```

This should be left blank so GitHub can compute the default

> Default: the repository’s default branch (usually master)